test/soca/gw/bkg2comrot.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+bindir=$1
+srcdir=$2
+
+# export env. var.
+source ${srcdir}/test/soca/gw/runtime_vars.sh $bindir $srcdir
+
+# prepare background from previous cycle
+mkdir -p $bindir/test/soca/gw/testrun/ROTDIRS/gdas_test/gdas.20180415/06/ocean/
```

Comment:
Use standard variables in the path here? like ROTDIR, PDY, cyc, etc

Reply:
Cleaned up the run time variables and now making use of pdy/cyc/...